presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -962,6 +971,7 @@ private Partition buildPartitionObject(Table table, PartitionUpdate partitionUpd
         else {
             partition.getStorageBuilder().setStorageFormat(fromHiveStorageFormat(defaultStorageFormat));
         }
+        partition.setParameters(ImmutableMap.of(PRESTO_VERSION_NAME, serverInfo.getVersion()));
```

Comment:
Use an `ImmutableMap` builder as done above for the table, so it will be easier to add more properties later